sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/models/ChatThreadProperties.java

```diff
@@ -9,9 +9,9 @@
 
 import org.threeten.bp.OffsetDateTime;
 
-/** The ChatThread model. */
+/** The ChatThreadProperties model. */
 @Fluent
-public final class ChatThread {
+public final class ChatThreadProperties {
```

Comment:
This class looks the same to the one under implemenation/models. Can we just move that one to models folder instead? It is just a simple change to the README.md in swagger folder.

Reply:
I think the one in impl based on swagger has the field `CommunicationIdentifierModel` which we don't want to expose to the users. This public type exposes that identifier as `CommunicationIdentifier` (from common).